modules/openshift-developer-cli-pushing-the-odo-init-image-to-a-mirror-registry.adoc

```diff
@@ -0,0 +1,119 @@
+
+[id="pushing-the-odo-init-image-to-a-mirror-registry_{context}"]
+= Pushing the `odo init` image to a mirror registry
+
+Depending on your operating system, you can push the `odo init` image to a cluster with a mirror registry as follows:
+
+[id="pushing-the-init-image-to-a-mirror-registry-on-linux_{context}"]
+
+== Pushing the `init` image to a mirror registry on Linux
+
+.Procedure
+
+. Use `base64` to encode the root certification authority (CA) content of your mirror registry:
++
+----
+$ echo <content_of_additional_ca> | base64 -d > disconnect-ca.crt
+----
+
+. Copy the encoded root CA certificate to the appropriate location:
++
+----
+$ sudo cp ./disconnect-ca.crt /etc/pki/ca-trust/source/anchors/<mirror-registry>.crt
+----
+
+. Trust a CA in your client platform and log into the {product-title} mirror registry:
++
+----
+$ sudo update-ca-trust enable && sudo systemctl daemon-reload && sudo systemctl restart / docker && docker login <mirror-registry>:5000 -u <username> -p <password>
+----
+
+. Mirror the `odo init` image:
++
+----
+$ oc image mirror registry.access.redhat.com/openshiftdo/odo-init-image-rhel7:<tag> <mirror-registry>:5000/openshiftdo/odo-init-image-rhel7:<tag>
+----
+
+. Override the default `odo init` image path by setting the `ODO_BOOTSTRAPPER_IMAGE` environment variable:
++
+----
+$ export ODO_BOOTSTRAPPER_IMAGE=<mirror-registry>:5000/openshiftdo/odo-init-image-rhel7:<tag>
+----
+
+
+[id="pushing-the-init-image-to-a-mirror-registry-on-macos_{context}"]
+
+== Pushing the `init` image to a mirror registry on MacOS
+
+.Procedure
+
+. Use `base64` to encode the root certification authority (CA) content of your mirror registry:
++
+----
+$ echo <content_of_additional_ca> | base64 -d > disconnect-ca.crt
+----
+
+. Copy the encoded root CA certificate to the appropriate location:
++
+.. Restart Docker using the Docker UI.
++
+.. Run the following command:
++
+----
+$ docker login <mirror-registry>:5000 -u <username> -p <password>
+----
+
+. Mirror the `odo init` image:
++
+----
+$ oc image mirror registry.access.redhat.com/openshiftdo/odo-init-image-rhel7:<tag> <mirror-registry>:5000/openshiftdo/odo-init-image-rhel7:<tag>
+----
+
+. Override the default `odo init` image path by setting the `ODO_BOOTSTRAPPER_IMAGE` environment variable:
++
+----
+$ export ODO_BOOTSTRAPPER_IMAGE=<mirror-registry>:5000/openshiftdo/odo-init-image-rhel7:<tag>
+----
+
+[id="pushing-the-init-image-to-a-mirror-registry-on-windows_{context}"]
+
+== Pushing the `init` image to a mirror registry on Windows
+
+.Procedure
+
+. Use `base64` to encode the root certification authority (CA) content of your mirror registry:
++
+----
+PS C:\> echo <content_of_additional_ca> | base64 -d > disconnect-ca.crt
+----
+
+. As an administrator, copy the encoded root CA certificate to the appropriate location by executing the following command in PowerShell:
```

Comment:
Either remove "in PowerShell" completely (because you already executed a command in it in the previous step), or move the "in PowerShell" to step 1 if you think it's important to call out. 

Same for the other procedure for Windows